modules/ROOT/pages/execution-plans/operators.adoc

```diff
@@ -2819,6 +2822,59 @@ Total database accesses: 2, total allocated memory: 64
 ======
 
 
+[[query-plan-assert-same-relationship]]
+== Assert Same Relationship
+// AssertSameRelationship
+
+The `AssertSameRelationship` operator is used to ensure that no relationship property uniqueness constraints are violated in the slotted and interpreted runtime.
+The example looks for the presence of a _works in_ relationship with the supplied duration and title, for a specific person and location.
+If part of the pattern can't be found, then it will be created.
```

Comment:
```suggestion
The example looks for the presence of a _works in_ relationship with the supplied duration and title, for a specific person and location.
```

I couldn't really make sense of this, because the query does not seem to be supplied with a duration or a title.

Reply:
I updated the data but did not update the description, _sigh_.
No wonder you couldn't make sense of it, it is nonsense!
Good catch, will update, thanks.